sys/linux/socket.txt

```diff
@@ -134,11 +135,26 @@ send_msghdr {
 	msg_flags	const[0, int32]
 }
 
+msghdr_sock {
+	msg_name	ptr[in, sockaddr_storage, opt]
+	msg_namelen	len[msg_name, int32]
+	msg_iov		ptr[in, array[iovec_in]]
+	msg_iovlen	len[msg_iov, intptr]
+	msg_control	ptr[in, array[cmsghdr_sock], opt]
+	msg_controllen	bytesize[msg_control, intptr]
+	msg_flags	const[0, int32]
+}
+
 send_mmsghdr {
 	msg_hdr	send_msghdr
 	msg_len	int32
 }
 
+mmsghdr_sock {
+	msg_hdr	msghdr_sock
+	msg_len	int32
```

Comment:
Since this is output data, I think we better use `const[0, int32]` type instead of just `int32` so that fuzzer does not waste iterations trying to pass different values here.
Please fix send_mmsghdr above in the same way too.